File: code_llm/zero-shot/gpt-4o-mini/Cubo Preto.cpp
#include <iostream>
using namespace std;

int main() {
    int N;
    cin >> N;

    int none = (N - 2) * (N - 2) * (N - 2);
    int one = 6 * (N - 2) * (N - 2);
    int two = 12 * (N - 2);
    int three = 8;

    cout << (none < 0 ? 0 : none) << endl;
    cout << (one < 0 ? 0 : one) << endl;
    cout << (two < 0 ? 0 : two) << endl;
    cout << (three < 0 ? 0 : three) << endl;

    return 0;
}

File: output/csbc_semish_2026/x_ai_grok_code_fast_1/few/python/Cubo Preto.py
import sys

data = sys.stdin.read().split()
N = int(data[0])

if N < 2:
    print(0)
    print(0)
    print(0)
    print(0)
else:
    zero = (N - 2) ** 3
    one = 6 * (N - 2) ** 2
    two = 12 * (N - 2)
    three = 8
    print(zero)
    print(one)
    print(two)
    print(three)